### PR TITLE
Clarify scan-result endpoint's body parameters

### PR DESCRIPTION
**Pull request by rtuin**
The scan-result POST body can use some specification of format etc. I've added a table with details to clarify the parameters.

Builds on #4

### CODE REVIEW

File: docs/endpoints/scan-results.md [hunk 1]
```diff
@@ -24,35 +24,63 @@ The body must contain all collected data from the client's device.
...
+| beaconLogs[].major      | The beacon's major number                                | yes      |
+| beaconLogs[].minor      | The beacon's minor number                                | yes      |
+| beaconLogs[].timestamp  | Timestamp of result in seconds                           | yes      |
+| beaconLogs[].timezone   | The client's timezone in UTC format                      | no       |
+| beaconLogs[].rssi       | The result's RSSI value                                  | yes      |
+| beaconLogs[].beacon_id  | The beacon's id as defined in Sensimity                  | no       |
+
 ### Response
-The response is a summary of the received results.
+The response is an indication of success. When the request was accepted, you will receive an HTTP-status of"204 No Content".
```
Comment (adri) [inline docs/endpoints/scan-results.md]: space between `of"204 No Content"` ;)

File: docs/endpoints/scan-results.md [hunk 2]
```diff
@@ -24,35 +24,63 @@ The body must contain all collected data from the client's device.
...
       "rssi": -60,
-      "id": 2,
-      "minor": 0,
       "beacon_id": 16
     }
   ]
 }
 ```
 
+| Parameter               | Description                                              | Required |
```
Comment (adri) [inline docs/endpoints/scan-results.md]: very nice!  :+1: 

Examples could be added to improve it even more.
The UTC timezone format can be also expressed like `UTC-01` (only hours) which is currently not accepted by the timezone validator. An example like `UTC+02:00` could help imo.

Comment (renatomefi) [inline docs/endpoints/scan-results.md]: I didn't get it, the current example is already "UTC+02:00"

File: docs/endpoints/scan-results.md [hunk 3]
```diff
@@ -24,35 +24,63 @@ The body must contain all collected data from the client's device.
...
+| instance_ref            | The application reference as you get from your dashboard | yes      |
+| device.model            | The client's device model                                | yes      |
+| device.operating_system | The client's device model                                | yes      |
+| device.version          | The client's device OS version                           | yes      |
+| device.device_id        | The client's unique device id                            | yes      |
+| beaconLogs[].UUID       | The beacon UUID                                          | yes      |
+| beaconLogs[].major      | The beacon's major number                                | yes      |
+| beaconLogs[].minor      | The beacon's minor number                                | yes      |
+| beaconLogs[].timestamp  | Timestamp of result in seconds                           | yes      |
+| beaconLogs[].timezone   | The client's timezone in UTC format                      | no       |
```
Comment (david4worx) [inline docs/endpoints/scan-results.md]: Just leaving this here for later if we ever decide to look on previous decisions; I still feel that the naming on this field is off. This should be timezoneOffset. Timezone (as seen in PHP) should be the actual timezone indicator in string form and must be one of the entries as seen in https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List in the tz column. That why (in PHP) the following

```
new \DateTimezone('UTC+02:00');
```

Will yield

```
Fatal error: Uncaught exception 'Exception' with message 'DateTimeZone::__construct(): Unknown or bad timezone (UTC+02:00)'
```

Also see https://en.wikipedia.org/wiki/Time_zone#Time_zones_and_time_offsets

edit; not blocking for this PR. This is just the documentation of what is currently implemented.

File: docs/endpoints/scan-results.md (hunk 2 above)
Comment (david4worx) [inline docs/endpoints/scan-results.md]: @adri not sure where you got

```
The UTC timezone format can be also expressed like UTC-01 (only hours)
```

but that is an invalid UTC timezone (offset) format. And as @renatomefidf said, the current example is already showing the expected format. Could you point out what needs to be improved?

Comment (rtuin) [inline docs/endpoints/scan-results.md]: Thanks guys, I've added an example

**Comment by renatomefi** [comment]
ready to merge @david4worx @adri @rtuin ?

**Pull request merged**